spec/controllers/grading_reviews_controller_spec.rb

```diff
@@ -7,7 +7,9 @@
   let(:term) { FactoryBot.create :term }
 
   describe 'GET #index' do
-    let!(:student_term_registration) { FactoryBot.create(:term_registration, :student, term: term) }
+    let!(:tutorial_group) { FactoryBot.create(:tutorial_group, term: term) }
+    let!(:student_group) { FactoryBot.create(:student_group, tutorial_group: tutorial_group) }
+    let!(:student_term_registration) { FactoryBot.create(:term_registration, :student, term: term, student_group: student_group) }
```

Comment:
Please also verify that the `email` and `student_group.title` are present in the view.